### PR TITLE
Add a section specifying docs format

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: docs/contributing/style-guide.rst [hunk 1]
```diff
@@ -18,9 +18,10 @@ It is recommended that contributors check for compliance by running flake8_. We
 
 Black
 -----
-All python code (except for the usually generated files in the migration folder) must be formatted according to the ruleset defined by black_.
-As black_ is able to automatically reformat python code, contributors are supposed to run `black .` in the repositories root directory.
-For various IDEs / editors, there is also an integration_ for black_.
+All python code (except for the usually generated files in the migration folder) must be formatted
+according to the ruleset defined by black_. As black_ is able to automatically reformat python code,
+contributors are supposed to run `black .` in the repositories root directory. For various IDEs /
```
Comment (bmbouter) [inline docs/contributing/style-guide.rst:23]:
```suggestion
contributors are supposed to run `black .` in the ``pulpcore`` root directory. For various IDEs /
```

**Comment by bmbouter** [comment]
This captures the current style expectations for pulpcore as I understand them. If you look at the overwhelming majority of existing docs I believe it aligns with this as well.

**Review by dralley — APPROVED** [review]

**Review by dralley — APPROVED** [review]

**Review by bmbouter — APPROVED** [review]

**Comment by bmbouter** [comment]
Maybe confirm before merging at the next pulpcore meeting?

**Comment by daviddavis** [comment]
I put it on the agenda.

**Review by goosemania — APPROVED** [review]

**Review by ipanova — APPROVED** [review]

**Pull request merged**